news/2023/2023-11-03-interview-with-ephemeral.md

```diff
@@ -0,0 +1,98 @@
+
+**What are your thoughts on the current state of medals?**
+
+Overall I think the past couple of releases, the [the last release](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in particular came out fantastic, I'm very happy with it. 
+I think they are getting better, but I think with operating at like maybe 20% of what medals will ultimately end up being when they're fully finished being implemented in lazer, because it is very limited what we can do with them at the moment. And lazer will pull back a lot of those limitations and criteria that realistically would not be possible in stable.
+
+**How do you come up with ideas for new medals?**
+
+I stare at a blank wall until they just appear in my head, that's how 90% of them are made.
+
+**Are there any plans for new types of medals?**
+
+We've planned to add multiplayer medals for a long time, and I have so many ideas for those. I will share one briefly, it's more or less a medal that is contagious, you catch it from other people by playing in multiplayer rooms with people that have the medal. I'm really excited for that one!
+
+**Are there any rejected medal ideas that you're willing to share?**
+
+Skylord almost ended up there actually, which some people probably might suggest that it should still be there, you know, because it's really-really hard. 
+There also was a beatmap pack medal that was meant to be for Porter beatmap pack, but I don't recall if we [signed](/wiki/People/Featured_Artists#selection) [Porter Robinson](https://en.wikipedia.org/wiki/Porter_Robinson). I think we almost had Porter aboard as a Featured Artist, but I think it fell through and we haven't really pursued it since.
+
+**What's your favourite medal?**
+
+Probably [Internment](https://osu.ppy.sh/home/news/2023-09-18-the-hunt-for-internment) at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.
+
+**Are there plans for challenge packs in other gamemodes?**
+
+Yep, absolutely. Once we get people willing to tutor us on the maps that would be good for that then yeah.
+
+**Any updates on the next batch of medals?**
+
+There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.
```

Comment:
```suggestion
There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to, and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.
```